h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -487,7 +487,9 @@ void checkAndInheritModelProperties() {
 
         if (require_consistent_training_frames) {
           if (trainingFrameRows < 0) trainingFrameRows = _parms.train().numRows();
-          Frame aTrainingFrame = aModel._parms.train(); // FIXME: can be null when aModel was imported
+          Frame aTrainingFrame = aModel._parms.train();
+          if (aTrainingFrame == null)
+            aTrainingFrame = aModel._output._cross_validation_holdout_predictions_frame_id.get();
```

Comment:
to avoid confusion, and as we're just concerned about how many rows were used to train the base models, can you please try to manipulate only rowNums instead of frames? 
Just to avoid this:
```
aTrainingFrame = aModel._output._cross_validation_holdout_predictions_frame_id.get();
```
as this is a purely ad-hoc statement that is not actually true.

Reply:
Good point! When I was writing it I was thinking about holdout_predictions as a part of training frame for metalearner so it didn't struck as odd.  😅